about.html

```diff
@@ -0,0 +1,99 @@
+            </div>
+            <p id="contact-statement">Please contact us via email for any further questions about our car club! info@subaruowners.org</p>
+        </div>
+    </header>
+    <section id="about" class="bg-primary">
+        <div class="container">
+            <div class="row">
+                <div class="col offset-lg-8 text-center mx-auto">
+                    <h2 class="text-white section-heading">Subaru Owners Club 2022 Logo</h2>
+                    <hr class="light my-4">
+                    <p class="text-faded mb-4">The logo of the Subaru Owners Club 2022 was decided through the logo competition from 8th June to 7th July. After online public, voting and selection committees's judging process, the logo from Naresh won the prize. After consultation with Naresh, we optimized the prize winning-logo for this year's summit as follows:</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="services">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 text-center">
+                    <h2 class="section-heading">See the past club meets</h2>
+                    <hr class="my-4">
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="row">
+                <div class="col-md-6" id="wheels-event">
+                    <h2>Wheels & Coffee</h2>
+<!--                    <img src="assets/img/thumbnails/gallery_01_11_thumb.jpg" style="box-shadow: 0px 0px rgb(33, 37, 41), 2px 0px;">-->
+                </div>
+                <div class="col-md-6" id="charity-run">
+                    <h2>Isiolo Charity Run</h2>
+<!--                    <img src="assets/img/thumbnails/gallery_01_10_thumb.jpg" style="text-shadow: 2px 0px;box-shadow: 2px 0px;text-align: center;">-->
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="portfolio" class="p-0"></section>
+    <section class="text-white bg-dark">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-6" id="cc">
+                    <p></p>
+                    <p id="cc1"><i class="fa fa-creative-commons"></i>&nbsp;creative commons kenya</p>
+                </div>
+                <div id="copyright" class="col-md-6">
+                    <p>2022 Creative Commons Kenya. Some Rights Reserved</p>
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Please use semantic HTML and wrap this section with `footer` tag
- [ ] Please decrease the padding inside the footer:
<img width="844" alt="Screenshot 2022-04-20 at 13 12 49" src="https://user-images.githubusercontent.com/77038610/164194236-0f9a8c99-65e6-4d22-a6ea-04abdb41cb58.png">

- [ ] In mobile view, please add partners section in about page, it should show before footer. 
- [ ] In mobile view, content in footer should not wrap, It  should contains a logo aligned to the left and text aligned to the right.

Your design  | Behance design
------------- | -------------
<img width="477" alt="Screenshot 2022-04-20 at 13 14 25" src="https://user-images.githubusercontent.com/77038610/164194577-19137811-ab70-4ee4-a204-f979c727be5e.png"> | <img width="451" alt="Screenshot 2022-04-20 at 13 14 09" src="https://user-images.githubusercontent.com/77038610/164194523-3b04f11a-8fc0-4f20-a4d8-f4c37f57092e.png">